calyx-opt/src/analysis/compute_static.rs

```diff
@@ -133,3 +134,90 @@ impl WithStatic for ir::While {
         Some(bound * b_time)
     }
 }
+
+pub trait IntoStatic {
+    type StaticCon;
+    fn make_static(&mut self) -> Option<Self::StaticCon>;
+}
+
+impl IntoStatic for ir::Seq {
+    type StaticCon = ir::StaticSeq;
+    fn make_static(&mut self) -> Option<Self::StaticCon> {
+        let mut static_stmts: Vec<ir::StaticControl> = Vec::new();
+        let mut latency = 0;
+        for stmt in self.stmts.iter() {
+            if !matches!(stmt, ir::Control::Static(_)) {
+                log::debug!("control under seq is not static");
```

Comment:
If these debug statements are meant to stay in the final PR, make the message more information.